# index-sorted-single-BCR-sequencing
This is a pipeline rundown of index-sorted sinlge cell sequencing of epitope specific memory B cells from PBMC or lymph nodes together with some follicular helpter T cells from clinical trial participants. The cells were index-sorted into 96-well plates and after the whole transcriptome transcripts were purified, reverse transcripted and amplified, both whole transcriptome and B cell receptor sequencing libraries were constructed via Nextera protocol, labeled with dual index (plate specific and well specific to define uniquity). The sequencing run was performed on Illumina Nextseq 1000 system wiht P1 300 cycle kit (151-10-15-151) and the fastq files were generated with the sequencer's on-board Dragon App.
The sequencer was mounted (https://knowledge.illumina.com/instrumentation/nextseq-1000-2000/instrumentation-nextseq-1000-2000-reference_material-list/000003037) to HPC for follow-up analysis. 
This Baldrlite pipeline was modified from Baldr with update IgBlast version By Fred Hutch Ju Yeong Kim with several parimeters changed by me to bypass the limit of job number on Biowulf and Skyline (HPC system for NIH and NIAID)
The Baldrlite results folder would have two .csv files, one is all the hits for all fastq submitted, the other filtered one only contains complete, productive, one top bowtie_read_count heavy and light chain.
After Baldrlite, all the other analysis scripts were generated by me for quality control. The sample info, such as time_point, participant_id, sample_type; and flow cytometry info, such as each gate's reading, clasification of epitope-specificity, cell_type should be merged together with sequencing data. To check whether there are shared clones between different epitope-specific cells, between donors, po
